### PR TITLE
doc: Python Plugin video and template links

### PR DESCRIPTION
**Pull request by hydratim**
#### Summary

Updates the Python Source Plugin page to add the Youtube Video and links to the Template Repo

### CODE REVIEW

**Review by erezrokah — APPROVED** [review]

**Review by hermanschaaf — COMMENTED** [review]
Nice!

**Comment by hydratim** [comment]
🎉

**Comment by hydratim** [comment]
Apparently, Adding the metadata doesn't auto-embed, so I'll have to add a second PR now

File: website/pages/docs/developers/creating-new-plugin/python-source.mdx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 title: Creating a New CloudQuery Source Plugin in Python
 description: Learn how to write a new CloudQuery source plugin from scratch in Python.
+image: "https://i3.ytimg.com/vi/TSbGHz5Z09M/maxresdefault.jpg"
```
Comment (erezrokah) [inline website/pages/docs/developers/creating-new-plugin/python-source.mdx:4]: What are we expecting these to do?

Comment (hydratim) [reply website/pages/docs/developers/creating-new-plugin/python-source.mdx:4]: I was expecting it to auto-embed the video, but that didn't work, so I'll follow this up with a second PR that manually adds the embed.

Comment (hydratim) [reply website/pages/docs/developers/creating-new-plugin/python-source.mdx:4]: https://github.com/cloudquery/cloudquery/pull/14782

**Pull request merged**